Yoast/yoast-components
# Render multi shape SVG icons passing an array of shapes to SvgIcon

## Summary

- Makes it possible to render multi-shape SVG icons

## Relevant technical choices:

- when the passed `path` is an array of shapes (`path`, `circle`, whatever), the shapes are rendered as the SVG content 
- for now, supports only simple multi shapes
- there's a bit of hardcoding with the color used for the secondary shapes, this could be improved in the future if need be

## Test instructions

- check the standalone app, yarn start and go to localhost:3333
- go in the SVG icons tab
- verify the seo-score-ok icon is rendered with an orange background and black eyes/mouth
- verify there are no errors

In the plugin:
- yarn link yoast-components
- build the JS
- edit a post and make the readability or SEO analysis give an OK result
- verify the icon is orange (see https://github.com/Yoast/wordpress-seo/pull/10603) with black eyes/mouth

I've tested it in all major desktop browsers.

Fixes https://github.com/Yoast/yoast-components/issues/702
